src/runtime/logging.cc

```diff
@@ -17,16 +17,15 @@
  * under the License.
  */
 
-#ifdef TVM_BACKTRACE_DISABLED
+#if TVM_USE_LIBBACKTRACE == 0
```

Comment:
I'd like to be consistent with other TVM #if usages.
```suggestion
#if TVM_USE_LIBBACKTRACE
```